sources/europe/Torokbalint-ortophoto-2013.json

```diff
@@ -0,0 +1,637 @@
+{
+    "url": "http://terkep.torokbalint.hu/mapproxy/service?FORMAT=image/jpeg&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=ORTO_2013_5CM_2013SZEPT_TAKARASSAL_512_512&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+    "license_url": "http://wiki.openstreetmap.org/wiki/WikiProject_Hungary/Ortofot%C3%B3k",
+    "attribution": {
+        "url": "http://www.torokbalint.hu/",
+        "text": "Törökbálint",
+        "required": true
+    },
+    "type": "wms",
+    "name": "Törökbálint ortophoto 2013",
+    "available_projections": [
+        "EPSG:4326",
+        "EPSG:900913",
```

Comment:
Hi @kolesar-andras 

For your information: I found that this WMS service doesn't support EPSG:900913 anymore (see #425), so I just removed it in 5eb9eea